<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeAnalyzeConfigDialog</class>
 <widget class="QDialog" name="CodeAnalyzeConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCM Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose the Target</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Choose a target project:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="projectSelectBox">
         <item>
          <property name="text">
           <string>Current project</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Project1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Project2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Project3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Project4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Choose target files:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fileSelectBox">
         <item>
          <property name="text">
           <string>All files</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>file 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>file 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>file 3</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Program Analysis Property Set Config</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,2,2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Choose a property set:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="setSelectBox">
         <item>
          <property name="text">
           <string>Default property set</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Property set 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Property set 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Property set 3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="manageButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Manage...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="ruleTree">
       <column>
        <property name="text">
         <string>Property Check Rules</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>NullPointerException</string>
        </property>
        <property name="checkState">
         <enum>PartiallyChecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>rule 1</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rule 2</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rule 3</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>IndexOutOfBoundsException</string>
        </property>
        <property name="checkState">
         <enum>PartiallyChecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>113: Thrown to indicate that an index of some sort  such as to an array, to a string, or to a vector is out of range</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rule 5</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>NumberFormatException</string>
        </property>
        <property name="checkState">
         <enum>PartiallyChecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>rule 6</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rule 7</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose the Other Techniques</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Automatic Test Generation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Automatic Testing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Check Requirements in the Target Specification</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,14,14">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>12</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Choose the target specification:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Current Specification</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Specification 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Specification 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Manage Specification</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>Check the Style of Coding</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Style of Code Configuration</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>Dynamicly Analyze the Code when programming</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startAnalysisButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
